### PR TITLE
[Docs] Change the process in all files to workflow

### PR DESCRIPTION
**Pull request by wushanru**
[Docs] Change the process in all files to workflow
A PR has been submitted before, closed by mistake, resubmit

### CODE REVIEW

File: docs/docs/en/DSIP.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Current DSIPs including all DSIP still work-in-progress, you could see in [curre
 
 Past DSIPs including all DSIP already done or retired for some reason, you could see in [past DSIPs][past-DSIPs]
 
-## DSIP Process
+## DSIP Workflow
```
Comment (zhongjiajie) [inline docs/docs/en/DSIP.md]: Wrong change
```suggestion
## DSIP Process
```

File: docs/docs/en/architecture/cache.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 ## Purpose
 
-Due to the large database read operations during the master-server scheduling process. Such as read tables like `tenant`, `user`, `processDefinition`, etc. Operations stress read pressure to the DB, and slow down the entire core scheduling process.
+Due to the large database read operations during the master-server scheduling workflow. Such as read tables like `tenant`, `user`, `processDefinition`, etc. Operations stress read pressure to the DB, and slow down the entire core scheduling workflow.
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/cache.md:5]: I think this content also should not change. Am I right @caishunfeng

File: docs/docs/en/architecture/design.md [hunk 3]
```diff
@@ -55,7 +55,7 @@
 
   - **WorkerManagerThread** is mainly responsible for the submission of the task queue, continuously receives tasks from the task queue, and submits them to the thread pool for processing;
 
-  - **TaskExecuteThread** is mainly responsible for the process of task execution, and the actual processing of tasks according to different task types;
+  - **TaskExecuteThread** is mainly responsible for the workflow of task execution, and the actual processing of tasks according to different task types;
```
Comment (zhongjiajie) [inline docs/docs/en/architecture/design.md]: personally think this should not change too

**Review by zhongjiajie — CHANGES_REQUESTED** [review]
maybe we should
* first replace `process instance` to `workflow instance`
* then replace `process definition` to `workflow definition`
* and then global search `process` to check whether is should be replaced or not

File: docs/docs/en/architecture/cache.md (hunk 2 above)
Comment (wushanru) [reply docs/docs/en/architecture/cache.md:5]: @caishunfeng Please confirm whether it needs to be modified

File: docs/docs/en/architecture/design.md (hunk 3 above)
Comment (wushanru) [reply docs/docs/en/architecture/design.md]: @zhongjiajie Please confirm whether it needs to be modified, thank you

Comment (EricGao888) [reply docs/docs/en/architecture/design.md]: No, shouldn't change it. Keep it as `process`.

Comment (wushanru) [reply docs/docs/en/architecture/design.md]: > No, shouldn't change it. Keep it as `process`.
OK, I understand

File: docs/docs/en/architecture/cache.md (hunk 2 above)
Comment (caishunfeng) [reply docs/docs/en/architecture/cache.md:5]: Yes,  shouldn't change it.

**Pull request closed without merging**